### ISSUE TITLE
Reports function missing get parameters

### ISSUE THREAD

**Issue by ibox4real** [XeroAPI/xero-node#223]
Hey, i am migrating to version 3  since we  are using typescript now. I have a problem with the reports api call where the only argument in the typescript files is ReportID. I want to put in additonal arguments but the compiler will start whining that it only accepts type `{ReportID: string}`

```ts
const PNL = xeroClient.reports.get({
    ReportID: 'ProfitAndLoss',
    fromDate: '2018-05-01',
    toDate: '2018-05-30'
})
```

**Comment by philals** [comment]
Hi @ibox4real 

Thanks for the issue. This is probably an issue with the code for this function. Looking at it, it only uses the ReportId. 

If you're able, please PR a fix to this file: https://github.com/XeroAPI/xero-node/blob/master/src/AccountingAPIClient.ts

Let me know if you need a hand.

**Comment by glenndevenish** [comment]
So could we we change [this](https://github.com/XeroAPI/xero-node/blob/master/src/AccountingAPIClient.ts#L922) to the below, or does the already optional `args?` cover the optional `fromDate` and `toDate`?

```typescript
get: async (args?: { 
  ReportID: string,
  fromDate?: string,
  toDate?: string
}): Promise<ReportsResponse> => {
```

**Comment by philals** [comment]
Thanks @glenndevenish and @ibox4real 

I have merged the PR and released as v3.0.9.

All the best!

**Comment by glenndevenish** [comment]
Hey @philals, I've made a [Xero report parser](https://gist.github.com/glenndevenish/036af94a5f22f23f27c96f5ce277b3f2)! 🎉 Hopefully someone might find it useful...
